source/usecases/usecase2/nodes.h5.txt

```diff
@@ -11,9 +11,9 @@ GROUP "/" {
                      CSET H5T_CSET_UTF8;
                      CTYPE H5T_C_S1;
                   }
-                  DATASPACE  SIMPLE { ( 2 ) / ( 2 ) }
+                  DATASPACE  SIMPLE { ( 1 ) / ( 1 ) }
                   DATA {
-                     "dNAC", "dSTUT"
+                     "dSTUT"
```

Comment:
I am not sure why this has changed (in usecase 1) too.

Reply:
I believe that this is due to samples being drawn randomly. In the usecase config we have
```yml
# from usecase_examples/config/nodes_configuration.yaml
etype:
  type: text
  values: [ "dNAC", "dSTUT" ]
```
and we only draw 2 samples, and since both of them were `"dSTUT"` only that value is represented in the `@library`.